src/reporters/errors/templates.js

```diff
@@ -14,6 +18,11 @@ function getMsgPrefix (err, category) {
     `;
 }
 
+function alignCode (str) {
+    //NOTE: we remove indentation before first line, because there is already in template.
```

Comment:
//NOTE: we remove indentation before the first line because the message template already contains the indent.